src/main/java/it/pagopa/pm/gateway/dto/transaction/UpdateAuthRequest.java

```diff
@@ -0,0 +1,22 @@
+package it.pagopa.pm.gateway.dto.transaction;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Data
+@AllArgsConstructor
```

Comment:
What about either:
* Removing all-args constructor; or
* Adding a constructor that takes a `ZonedDateTime` instead of a `String`; or
* Using a `ZonedDateTime` field instead of a `String`
?

Reply:
The ZonedDateTime converted to string doesn't match the format needed by the service.
This would be the outcome `2022-12-21T17:20:41.330+01:00[Europe/Berlin]` while the swagger says this one is expected `2022-02-11T13:00:00+01:00`.

I'm creating a construtor that takes a ZonedDateTime and then formats it internally.

Reply:
What about `OffsetDateTime`?

Reply:
This one would be the result: `2022-12-22T10:48:17.950+01:00`. Which is closer but not correct because contains the milliseconds as well.